logprep/connector/confluent_kafka/output.py

```diff
@@ -138,6 +137,13 @@ class Metrics(Output.Metrics):
         )
         """Total number of message bytes (including framing, such as per-Message framing and
         MessageSet/batch framing) transmitted to Kafka brokers"""
+        number_of_successfully_delivered_events: CounterMetric = field(
+            factory=lambda: CounterMetric(
+                description="Number of events that were successfully delivered to Kafka",
+                name="number_of_successfully_delivered_events",
+            )
+        )
+        """Number of events that were successfully delivered to Kafka"""
```

Comment:
yes ok it is not your issue now, but could you please add the documentation for the other kafka_output config parameters?

Reply:
I have added some documentation now.